drivers/cache/Kconfig.nrfx

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2022 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config CACHE_NRFX_CACHE
+	bool "NRFX cache driver"
+	select CACHE_HAS_DRIVER
+	depends on SOC_SERIES_NRF54LX
```

I'd suggest to change this to:
```suggestion
	depends on HAS_NRFX && (CPU_HAS_ICACHE || CPU_HAS_DCACHE)
```